lib/entry.ts

```diff
@@ -16,6 +16,7 @@ export default function createEntry(
   entryData: any,
   fieldInfo: EntryFieldInfo[],
   createEntryField: (info: EntryFieldInfo) => ExhaustiveEntryFieldAPI,
+  release?: Release,
```

Comment:
I think we should not depend on the release itself (acting as a "scope") to determine whether entryData is/is not a "release entry". this information is "extrinsic" to the function itself -- meaning all it takes is for someone to call this function incorrectly and it'll falsely assume the publish/unpubllsh are not valid.

Reply:
After chatting with Tolkien w've learned that release entries are indistinguishable from regular entries in the data themselves, and we rely on passing / preserving release context to make this distinction wherever it's needed.

So I take back the earlier comment and we can proceed with the general approach.